src/conventional-commits.ts

```diff
@@ -81,6 +85,9 @@ function getParsedCommits(commits: Commit[]): CommitWithHash[] {
         const commitWithHash = postProcessCommits(
           parsedCommit
         ) as CommitWithHash;
+        if (commitFilter?.(parsedCommit)) {
```

Comment:
what if we default the `commitFilter` to a filter that always returns `true`, the current behavior, then we don't need to use this optional call?

Reply:
Defaulted to false, as that will allow a commit to be processed.